src/pages/home/home.styled.ts

```diff
@@ -12,6 +12,24 @@ export const CONTENT_CLASSES = {
   }
 };
 
+export const AFFILIATE_CLASSES = `${FLEX_CENTER} mb-4 mb-lg-0`;
+export const TESTIMONIAL_CLASSES = `${FLEX_CENTER} mb-6 mb-lg-0`;
+
+export const AFFILICATION_CONFIG_CLASSES = [
+  AFFILIATE_CLASSES,
```

Comment:
this and below using fixed sizes is not a good practice. If a programmer adds a data entry in the data array, they have to know to add it here too.

Can you use `length` to generate the classes?

also, is `AFFILICATION` a typo, not sure what this word means

Reply:
im not too sure how to fix this issue then?  it's going to be the same issue as if the programmer adds 4 cards manually and adds the classes on each card, they'll have to know to style the card

Reply:
used a custom class instead of class array